lib/waterline/utils/ontology/is-capable-of-optimized-populate.js

```diff
@@ -115,17 +114,17 @@ module.exports = function isCapableOfOptimizedPopulate(attrName, modelIdentity,
   //
   // (remember, we just checked to verify that they're exactly the same above-- so we could have grabbed
   // this datastore name from ANY of the involved models)
-  var relevantDatastoreName = PrimaryWLModel.datastore;
```

Comment:
This section seems to have gotten a little smashed around-- needs to be looked at holistically